ci/pbs/run-master.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Reduce time between PBS scheduling
+qmgr -c "set server scheduler_iteration = 20"
+# add two slaves to pbs
+qmgr -c "create node pbs_slave_1"
+qmgr -c "create node pbs_slave_2"
+
+# wait until the end of tests
+/bin/sleep 3600
```

Comment:
The two are working. The idea is to have a blocking process so as not to exit the docker run command, and have the container always up during the test.
I just found it weird to start a Python HTTP server just to lock the launching script in a process, but this is a perfectly working solution, which would last longer than mine :slightly_smiling_face: .

Idealy, we should block on one of the PBS process, or SGE in the other case.

Reply:
> I just found it weird to start a Python HTTP server just to lock the launching script in a process, but this is a perfectly working solution, which would last longer than mine

OK this may show my sheer ignorance of docker-related things, but my worry is what happens if I start the cluster locally on my laptop, forget about it and come back the next day, will I still be able to do things like `docker exec -it pbs_master /bin/bash -c`

Reply:
> will I still be able to do things like docker exec -it pbs_master '/bin/bash -c'

Nope, you won't! Cluster will shut down after 1 hour as you expected.

I get your point here too. I don't really like the idea of starting a fake python HTTP server just to have a hanging process, but I don't really have time to look for other solutions, so lets do it this way 😁 .